packages/ckeditor5-ui/src/dropdown/utils.ts

```diff
@@ -38,7 +38,6 @@ import {
 } from '@ckeditor/ckeditor5-utils';
 
 import '../../theme/components/dropdown/toolbardropdown.css';
-import '../../theme/components/dropdown/listdropdown.css';
```

Comment:
We no longer need to import the empty CSS files, because the entire theme-lark is built as one CSS file. This increases the base CSS size by about 10kB gzipped, but we no longer need CKEditor specific plugins for webpack or Vite.

Reply:
Note that we need to keep these empty CSS files if we want to still support old installation method.